apacheconfig/wloader.py

```diff
@@ -30,98 +28,149 @@ class Node():
 
     @abc.abstractmethod
     def __str__(self):
-        raise NotImplementedError()
+        """Writes this node to a raw string. To get more metadata about the
+        object, use ``repr``.
+        """
+        pass
```

Comment:
nit: in presence of a docstring, `pass` can be omitted

nit++: both `__str__` and `__repr__` have predefined semantics: the first prints the object prettily, the second (ideally) produces valid Python code which can be evaluated and re-produce a copy of the original object. 

That gives me the impression that:

1. it sometimes makes sense to stick with this pre-defined meanings whenever possible
2. if (1) holds, the docstring is not really necessary because everyone expects what these methods would do

Tell me when you can't tolerate my nitpicking any longer!

Reply:
These are great comments! I'm not fully familiar with Python best practices, so they're very helpful. What I'll do here then is:

1. Move current functionality of `str` to a new function called `dump()`
2. Move functionality of `__repr__` to `__str__`
3. Remove docstsring